CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ guide](./docs/migration-guide.md) for details.
 
 - [ENHANCEMENT] Add the option to log to stdout instead of a Loki instance. (@joe-elliott)
 
+- [ENHANCEMENT] Update Prometheus dependency to v2.27.0 and add exemplars support (@mapno)
```

Comment:
Can you also note the dependency bumps for Cortex and Loki/Promtail?

Reply:
I wasn't sure on how to mention this, as it's using unreleased commits for both projects.

Reply:
This is how I've done it before: 

> [ENHANCEMENT] Update Loki to 59a34f9867ce. This is a non-release build, and was needed to support multiple Loki instances. (@rfratto)